.github/workflows/.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] This file should be located in the root folder, please delete this file from this directory.